sim/CMakeLists.txt

```diff
@@ -4,8 +4,19 @@ project(bentobox)
 # Source files
 set(SRC_DIR "${CMAKE_CURRENT_SOURCE_DIR}/src")
 set(LIB_DIR "${CMAKE_CURRENT_SOURCE_DIR}/lib")
-set(SOURCES "${SRC_DIR}/main.cpp")
-set(TESTS "${SRC_DIR}/test.cpp")
+set(ICS_DIR "${SRC_DIR}/ics")
+set(UTIL_DIR "${SRC_DIR}/util")
+
+set_property(GLOBAL PROPERTY hello "hello world")
```

Comment:
Is this property used?

Reply:
No, was testing out some CMake stuff, I forgot to remove it

Reply:
I have just pushed a commit to remove it